### ISSUE TITLE
Validation split for training set

### ISSUE THREAD

**Issue by Voldemort108X** [NVIDIA/pix2pixHD#228]
Hi,

I am wondering whether pix2pixHD supports a validation split in the training set. I notice that the model's fitting on the training set is very good but its generalization on the testing set is limited on my customized dataset (perhaps overfitting), while both training and testing share a very similar distribution. I also tried to reduce the number of resnet blocks in the global generator (9,6,3), increase the batch size (1,4,8). All the experiments I tried have a similar issue. I suspect it is the lack of validation set in training that results in such an issue. 

Thanks to anyone who answers my question.

P.S. Training set details: 1024x1024 with 1042 images. Testing set: 87 images of the same size. I think the number is enough to learn the distribution.
